score/ScoreBonus.py

```diff
@@ -0,0 +1,275 @@
+from nameinternal import get_card_metadata, get_score_bonus
+from collections import Counter, namedtuple
```

Comment:
Please put built-in imports above process-level imports (with a newline between), as a style preference!